### PR TITLE
feat: new login page

### PR DESCRIPTION
**Pull request by v0l**

### CODE REVIEW

File: src/Pages/Login.tsx [hunk 1]
```diff
@@ -94,38 +136,88 @@ export default function LoginPage() {
...
-      <>
-        <h2>Other Login Methods</h2>
-        <div className="flex">
-          <button type="button" onClick={doNip07Login}>
-            Login with Extension (NIP-07)
-          </button>
-        </div>
-      </>
+      <button type="button" onClick={doNip07Login}>
+        Login with Extension (NIP-07)
```
Comment (verbiricha) [inline src/Pages/Login.tsx:140]: this should also be translated, no?

File: src/Pages/Login.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.login .login-actions > button {
+  margin-right: 10px;
+}
+
+.login .login-or {
+  margin-top: 56px;
+  margin-bottom: 64px;
+}
```
Comment (verbiricha) [inline src/Pages/Login.css:83]: I'd add a small margin to the left of the zap button:

```suggestion
}

.login .artwork .zap-button { 
  margin-left: 6px; 
}
```

**Review by verbiricha — APPROVED** [review]
minor comments, lgtm 🔥

File: src/Pages/Login.css (hunk 2 above)
Comment (v0l) [reply src/Pages/Login.css:83]: Didnt add margin, but removed the BG color so it looks like there is some spacing instead.

**Pull request merged**